zabartcc/ui
# ZAB Release 1.12

## ZAB Release

---

### Summary

Full changelog
1. Added VATUSA Traning record submission.
2. Changed 404 page
3. Fix small bug in staff page

---
### Testing

1. Open Staff page look for any error
2. Submit a traning record
3. check the 404 page
---

### Post-Deployment Validation

How to verify in the production environment.

---
